packages/chips/package.json

```diff
@@ -22,9 +22,13 @@
     "@material/chips": "^1.1.1",
     "@material/react-ripple": "^0.12.0",
     "classnames": "^2.2.6",
-    "react": "^16.4.2"
+    "react": "^16.4.2",
+    "react-is": "^16.8.6"
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@types/react-is": "^16.7.1"
```

Comment:
we should also have @types/react here as well. I have an issue to add it to all components, but until we get to that let's add it here too.

Reply:
done!

Reply:
I added to all packages in #898

Reply:
oh nice!